packages/matter.js/src/model/aspects/Constraint.ts

```diff
@@ -202,8 +202,8 @@ export namespace Constraint {
 
     function parseValue(numOrName: string): FieldValue {
         let value;
-        if (numOrName.startsWith("0x") || numOrName.startsWith("0b")) {
-            value = Number.parseInt(numOrName);
+        if (numOrName.match(/^-?0[xb]/)) {
```

Comment:
When I remember correctly an analysis i once saw then in general startsWith is faster then a regex ...

Reply:
I added sign support so it would've become 4x startsWith

Reply:
kk